parts/language/lang_en.lua

```diff
@@ -104,36 +104,36 @@ return{
 	chatHistory="------New messages below------",
 
-	noRooms="No Rooms Now",
-	roomsCreateFailed="Failed to create room",
-	roomsFetchFailed="Failed to fetch rooms",
+	noRooms="Such emptiness much void, come back later?",
+	roomsCreateFailed="Failed to create room.",
+	roomsFetchFailed="Failed to fetch room list.",
 
-	errorMsg="An error has occurred and Techmino needs to restart.\nError info has been created, and you can send it to the author.",
+	errorMsg="Techmino ran into a problem and needs to restart.\nError info has been logged, and you can send it to the devs.",
 
-	modInstruction="Select mods (modifiers) to use!\nMods will modify the game in different ways\n(and possibly break the game in weird ways).\nSome mods will make your game unranked.",
+	modInstruction="Select modifiers!\nMods modify the game in different manners.\n(and possibly break it in weird ways).\nBeware! Some mods unrank your game.",
 	modInfo={
-		next="Next\nOverride the Next queue length",
-		hold="Hold\nOverride the Hold queue length",
-		hideNext="Hidden Next\nHide a few upcoming Next preview pieces",
-		infHold="Infinite Hold\nYou can use the Hold function infinitely.",
-		hideBlock="Hide current block:\nMake the piece you are controlling invisible",
-		hideGhost="Hide ghost\nMake ghost invisible",
-		hidden="Hide dropped blocks\nPieces fade disappear after lockdown",
-		hideBoard="Hide board\nHide part of the board, or the entire board",
-		flipBoard="Flip board\nFlip or rotate the board.",
-		dropDelay="Falling speed\nOverride falling speed (unit: frames/block)",
-		lockDelay="Lockdown delay\nOverride lockdown delay (unit: frames)",
-		waitDelay="Spawn delay\nOverride spawn delay (unit: frames)",
-		fallDelay="Line delay\nOverride line delay (unit: frames)",
-		life="Life\nOverride number of lives players begin with",
-		forceB2B="B2B only\nGame ends when B2B gauge drops below startup line",
-		forceFinesse="Finesse only\nGame ends on finesse fault",
-		tele="Teleport\nForce 0 DAS and 0 ARR movement",
-		noRotation="No rotation\nRotation buttons are disabled",
-		noMove="No movement\nLeft and right movement buttons are disabled",
-		customSeq="Randomizer\nOverride the randomizer for the game",
-		pushSpeed="Garbage speed\nOverride the speed at which garbage lines rise (unit: blocks/frame)",
-		boneBlock="Bone\nPlay with the bone block skin",
+		next="NEXT\nOverride the how many NEXTs you see.",
+		hold="HOLD\nOverride the how much HOLD you use.",
+		hideNext="Hidden NEXT\nDecide how many of NEXTs are hidden.",
+		infHold="InfiniHold\nYou can use HOLD as much as you want!",
+		hideBlock="Hide Current Piece:\nMake the piece you are controlling invisible.",
+		hideGhost="No Ghost\nLiterally. No ghost.",
+		hidden="Hide Locked Pieces.\nPieces fade after locking.",
+		hideBoard="Hide Board\nHide part or the entire matrix.",
+		flipBoard="Flip Board\nFlip or rotate the matrix.",
+		dropDelay="Gravity\nOverride falling speed. (unit: frames/block)",
```

Comment:
Falling speed is not "gravity" - Gravity refers to the game mechanic where unsupported blocks can fall down, like in Tetris Journey

Reply:
https://tetris.fandom.com/wiki/Drop#Gravity

Reply:
~~Um, OK then, I didn't expect you were using non-Guideline terms~~

Anyways TTC doesn't use "gravity" like that

Reply:
I suppose that *avoid* using Guideline terms while having a bit of consistency is a better choice for copyright purposes, tho I guess you're fine this way.